docs/source/concepts.rst

```diff
@@ -266,22 +266,15 @@ These events could be used to attach any handler and are fired using :meth:`~ign
 If you want to use filtering with custom events (e.g. ``CustomEvents.CUSTOM_STARTED(every=5)``), you need to do 3 more things:
 
 - ``engine.state`` should have corresponding attributes for the events, e.g. ``engine.state.custom_started``
-- you need to pass a dict `event_to_attr` to :meth:`~ignite.engine.engine.Engine.register_events`, which maps between events and state attributes, e.g.
+- you need to pass a dict `attr_to_events` to :meth:`~ignite.engine.engine.Engine.register_events`, which maps between state attributes and a list of events, when setting the attribute to any value, all the counters for the events in the corresponding list will be updated accordingly. For getting the attribute value, the counter value of the first event in the corresponding list will be returned automatically.
 
 .. code-block:: python
 
-    event_to_attr = {
-        CustomEvents.CUSTOM_STARTED: "custom_started",
-        CustomEvents.CUSTOM_COMPLETED: "custom_completed",
+    attr_to_events = {
+        "custom_started": [CustomEvents.CUSTOM_STARTED],
+        "custom_completed": [CustomEvents.CUSTOM_COMPLETED]
```

Comment:
An explicit example is necessary to illustrate the explanation and how things are working